AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -202,6 +202,15 @@ - (void)willRemoveSubview:(UIView *)subview
 #endif
 }
 
+- (CGSize)sizeThatFits:(CGSize)size
```

Comment:
Yes it does not immediately change the size of this node. What `layoutThatFits:` doesn't do either. Unfortunately `layoutThatFits:` has indeed the side effect of creating a pending layout, but it actually only applies the pending layout in the layout pass if the size of the `bounds` is the same as the pending layout size. So in theory it should work out.